README.md

```diff
@@ -76,6 +75,43 @@ You can find more options in `sqllogictest --help`.
 
 Note that only postgres is supported now.
 
+## `.slt` Test File Format Cookbook
```

Comment:
You can see a rendered preview of what this would look like at https://github.com/alamb/sqllogictest-rs/tree/alamb/docs#slt-test-file-format-cookbook